staging/kos/pkg/apis/network/types.go

```diff
@@ -178,9 +178,34 @@ type SubnetSpec struct {
 }
 
 type SubnetStatus struct {
-	// Errors are the complaints, if any, from the IPAM controller.
+	// Validated tells users and consumers whether the subnet spec has passed
+	// validation or not. The fields that undergo validation are VNI and CIDR.
+	// As a consequence, if Validated is true it is guaranteed to stay true
```

Comment:
The latch to `true` is a consequence of more than the aforementioned fact.  It is also a consequence of the fact that conflicts with others are not re-considered once `Validated == true`.  I suggest s/As a consequence, if/If/.

Reply:
Fixed in 8ac9883.